File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportJava/Networking/src/main/java/transport/networking/dto/ReservationDTO.java
package transport.networking.dto;

import java.io.Serializable;

public class ReservationDTO implements Serializable {
    private Long id;
    private String clientName;
    private Long tripId;
    private int numberOfSeats;

    public ReservationDTO(Long id, String clientName, Long tripId, int numberOfSeats) {
        this.id = id;
        this.clientName = clientName;
        this.tripId = tripId;
        this.numberOfSeats = numberOfSeats;
    }

    public Long getId() {
        return id;
    }

    public String getClientName() {
        return clientName;
    }

    public Long getTripId() {
        return tripId;
    }

    public int getNumberOfSeats() {
        return numberOfSeats;
    }
}

File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportC#/Network/dto/ReservationDTO.cs
using System.Text.Json.Serialization;

namespace FirmaTransportC_.Networking.dto
{
    [Serializable]
    public class ReservationDTO
    {
        public long Id { get; set; }
        public long TripId { get; set; }
        public int NumberOfSeats { get; set; }
        public string ClientName { get; set; }

        public ReservationDTO(long id, long tripId, int numberOfSeats, string clientName)
        {
            Id = id;
            TripId = tripId;
            NumberOfSeats = numberOfSeats;
            ClientName = clientName;
        }

        public ReservationDTO()
        {
        }
    }
}
